src/components/feed/AnswerList.js

```diff
@@ -40,16 +49,24 @@ export const AnswerList = ({
                     </div>
                 </div>
             ) : (
-                <div className="Questions-container noQuestion">
-                    <span className="Questions-numberOfQuestions">
-                        <img src={messageIconBrown} alt="messageIconBrown" />{" "}
-                        아직 질문이 없습니다
-                    </span>
-                    <img
-                        className="Questions-emptyMessageIcon"
-                        src={emptyMessageIcon}
-                        alt="emptyMessageIcon"
-                    />
+                <div className="answer-container">
+                    <div className="answer-top-wrapper">
+                        <button onClick={handleDeleteFeed}>삭제하기</button>
+                    </div>
+                    <div className="Questions-container noQuestion">
+                        <span className="Questions-numberOfQuestions">
+                            <img
+                                src={messageIconBrown}
+                                alt="messageIconBrown"
+                            />{" "}
+                            아직 질문이 없습니다
+                        </span>
+                        <img
+                            className="Questions-emptyMessageIcon"
+                            src={emptyMessageIcon}
+                            alt="emptyMessageIcon"
+                        />
+                    </div>
```

Comment:
이 부분 제가 아까 NoQuestion 컴포넌트로 따로 분리해둬서,
이따 다같이 리팩토링할 때 드래그한 부분 전부 삭제하고 아래처럼 변경하면 될 것 같아요!

```suggestion
<NoQuestion />
```